component/src/main/java/org/wso2/carbon/identity/authenticator/bitly/BitlyAuthenticator.java

```diff
@@ -263,4 +273,43 @@ protected Map<ClaimMapping, String> getSubjectAttributes(OAuthClientResponse tok
         }
         return claims;
     }
+
+    @Override
+    protected String sendRequest(String url, String accessToken) throws IOException {
+
+        if (log.isDebugEnabled()) {
+            log.debug("Claim URL: " + url);
```

Comment:
What does it mean by the claim URL ?

Reply:
If so, can we consider adding the authenticator identifier to the debug logs ?

ex: `log.debug("Bitly user info response: " + builder.toString());`

Reply:
We can modify this debug log to show `log.debug("URL: " + url)`.

Reply:
Fixed